ui/app/components/clients/page/counts.hbs

```diff
@@ -151,32 +151,7 @@
         </Hds::Alert>
       {{/if}}
 
-      <nav class="tabs has-bottom-margin-s" aria-label="navigation for managing client counts">
```

Comment:
this file & its associated test were getting a bit hairy, so moved it into a separate component 🧹